### ISSUE TITLE
setNetworkConnection issue on Appium 1.5.3

### ISSUE THREAD

**Issue by hectorv** [appium/appium#6658]
## The problem

Hi! I'm trying to upgrade our test suite from Appium 1.4.16 to 1.5.3. Everything works fine except one test where we put the app in Airplane mode.

## Environment

* Appium version (or git revision) that exhibits the issue: git tag `1.5.3` (from source)
* Last Appium version that did not exhibit the issue (if applicable): `1.4.16`
* Desktop OS/version used to run Appium: OS X 10.11.4
* Node.js version (unless using Appium.app|exe): `v5.10.1`
* Mobile platform/version under test: Android 4.4.4 on Genymotion
* Real device or emulator/simulator: Emulator
* Appium CLI or Appium.app|exe: CLI

## Details

Describe in more detail the problem you have been experiencing, if necessary.

## Link to Appium logs

This is the error in the Appium server:
https://gist.github.com/hectorv/68622c55be23e7a0f60373a4e62af07c

and this is the error in the Python client:
```
In [3]: self.driver.set_network_connection(6)

/path/to/.virtualenvs/duo/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.pyc in execute(self, driver_command, params)
    234         response = self.command_executor.execute(driver_command, params)
    235         if response:
--> 236             self.error_handler.check_response(response)
    237             response['value'] = self._unwrap_value(
    238                 response.get('value', None))

/path/to/.virtualenvs/duo/lib/python2.7/site-packages/appium/webdriver/errorhandler.pyc in check_response(self, response)
     27                 raise NoSuchContextException(wde.msg, wde.screen, wde.stacktrace)
     28             else:
---> 29                 raise wde
     30

WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Cannot set property 'ignoreUnexpectedShutdown' of undefined
```

Any ideas what could be going on here? 
Thanks!

**Comment by JaniJegoroff** [comment]
There are couple of more issues regarding setting network connection mode that may give you more info:
https://github.com/appium/appium/issues/6615
https://github.com/appium/appium/issues/6625

**Comment by hectorv** [comment]
Here's some additional information. Even though `toggle_airplane_mode`[[1]](https://github.com/appium/appium-base-driver/blob/master/docs/mjsonwp/protocol-methods.md) isn't currently in the Python client, I called it directly like so:

```python
In [40]: base_url = 'http://127.0.0.1:4723/wd/hub/session/{}/appium/device/toggle_airplane_mode'
In [41]: url = base_url.format(self.driver.session_id)
In [42]: url
Out[42]: 'http://127.0.0.1:4723/wd/hub/session/f9e5fc0f-3711-4940-a024-2ad856c2dab0/appium/device/toggle_airplane_mode'
In [43]: r = requests.post(url)
In [44]: r.ok
Out[44]: False
In [46]: r.status_code
Out[46]: 500
In [47]: r.reason
Out[47]: 'Internal Server Error'
In [48]: r.json()
Out[48]:
{u'sessionId': u'f9e5fc0f-3711-4940-a024-2ad856c2dab0',
 u'status': 13,
 u'value': {u'message': u"An unknown server-side error occurred while processing the command. Original error: Cannot set property 'ignoreUnexpectedShutdown' of undefined"}}
```

and this is what I got on the server side:
https://gist.github.com/hectorv/8423281415e419684a5c7278ecd06125

**Comment by imurchie** [comment]
This ought to be fixed by `appium-selendroid-driver@1.3.5`. If you are running from source, delete `node_modules` and re-run `npm install`. If you are running from `npm`, re-install appium with `npm install --no-shrinkwrap appium`.

**Comment by hectorv** [comment]
Thanks @imurchie! I just verified the fix locally and can confirm that it works. Now, assuming that this fix will make it into the next release (1.5.4), do you have an idea when it'll be available on Sauce Labs?
